XamlControlsGallery/XamlControlsGallery.sln

```diff
@@ -1,10 +1,15 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2027
+# Visual Studio Version 16
```

Comment:
Why are we excluding Visual Studio 2017 here?

Reply:
I don't believe that excludes earlier versions, as that is set by `MinimumVisualStudioVersion = 10.0.40219.1`.

My VS just modified those lines automatically when the `.editorconfig` was added to the solution.

Reply:
I guess if they don't mean anything (from what I could find online), we can leave it that way.

Reply:
I actually tried manually changing them back, but then VS fought against me and won by flipping them again before the commit was made...